wscript

```diff
@@ -317,7 +317,13 @@ def _check_mac_osx_version(floor_version):
         return None
 
     # Extract the integer values between the '.'s
-    osx_version_major, osx_version_minor, osx_version_patch = tuple(int(x) for x in s.strip().split('.'))
+    osx_version_data = tuple(int(x) for x in s.strip().split('.'))
+    osx_version_major = osx_version_data[0]
+    osx_version_minor = osx_version_data[1]
+    osx_version_patch = 0
+
+    if len(osx_version_data) > 2:
+        osx_version_patch = osx_version_data[2]
```

Comment:
```suggestion
    if len(osx_version_data) > 2:
        osx_version_patch = osx_version_data[2]
    else:
        osx_version_patch = 0
```

Reply:
Both solutions look fine though I prefer the JH solution as the variable has a value by default.